RubeesEat/wwwroot/css/site.css

```diff
@@ -0,0 +1,172 @@
+/*source: https://www.joshwcomeau.com/css/custom-css-reset/*/
+*, *::before, *::after {
+    box-sizing: border-box;
+}
+
+* {
+    margin: 0;
+    font-family: "Metric Regular", Arial, Tahoma, sans-serif;
+    text-align: center;
+}
+
+body {
+    line-height: 1.5;
+    -webkit-font-smoothing: antialiased;
+}
+
+img, picture, video, canvas, svg {
+    display: block;
+    max-width: 100%;
+}
+
+input, button, textarea, select {
+    font: inherit;
+}
+
+p, h1, h2, h3, h4, h5, h6 {
+    overflow-wrap: break-word;
+    text-align: center;
+}
+
+#root, #__next {
+    isolation: isolate;
+}
+
+body {
+    font-family: Arial, sans-serif;
+    margin: 0;
+    padding: 0;
+    background-color: #2d332f;
+    color: white;
+}
+
+nav ul {
+    list-style-type: none;
+    padding: 0;
+    margin: 0;
+    display: flex;
+    align-items: center;
+}
+
+nav ul li {
+    margin-right: 1.5em;
+    transition: all 0.3s ease;
+}
+
+nav ul li a {
+    color: black;
+    text-decoration: none;
+    font-weight: bold;
+}
+
+nav ul li:hover {
+    transform: scale(1.15);
+    z-index: 1;
+}
+
+.logo {
+    width: 50px;
+    height: 50px;
+    margin-left: auto;
+}
+
+nav {
+    background-color: #fbbf00;
+    padding: 1em 2em;
+    box-shadow: 0 2px 5px rgba(0, 0, 0, 0.2);
+    position: sticky;
+    top: 0;
+    z-index: 1000;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.nav-container {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    max-width: 1200px;
+    margin: 0 auto;
+}
+
+.logo-container {
+    display: flex;
+    align-items: center;
+}
+
+.logo {
+    width: 50px;
+    height: 50px;
+}
+
+.nav-menu {
+    display: flex;
+    list-style-type: none;
+    padding: 0;
+    margin: 0;
+}
+
+.nav-menu li {
+    margin-right: 1.5em;
+}
+
+.nav-menu li a {
+    color: black;
+    text-decoration: none;
+    font-weight: bold;
+}
```

Comment:
Instead of adding a class of the list, we can also do `nav > ul`. This way, we don't have to introduce a class for each subelement of the root control (the `nav`).